cli/cmd/devtool/data/cloud-deps.docker-compose.yml

```diff
@@ -1,37 +1,43 @@
+# Service template
+x-service: &service
+  restart: always
+
+# Postgres template
+x-postgres: &postgres
```

Comment:
This is subjective and up to you, but I actually found the previous compose file more readable, even if there was a bit of duplication

Reply:
I don't mind either way ~ just a preference for me